airbyte-integrations/connectors/destination-e2e-test/src/main/resources/spec.json

```diff
@@ -1,6 +1,8 @@
 {
   "documentationUrl": "https://example.com",
   "supportsIncremental": true,
+  "supportsNormalization": true,
```

Comment:
I was not sure about `destination-e2e-test` spec

Reply:
let's default to false for both